Automattic/woocommerce-payments
# Place Woopay “Direct Checkout” Behind a Feature Flag

Fixes 2445-gh-Automattic/woopay

#### Changes proposed in this Pull Request

- Place the "direct checkout" feature behind a `_wcpay_feature_woopay_direct_checkout` option feature flag.

#### Testing instructions

1. Checkout to this branch and 2390-gh-Automattic/woopay.
2. Add a new option: `_wcpay_feature_woopay_direct_checkout` set to `1`.
3. Follow the instructions in 2390-gh-Automattic/woopay and ensure it tests as expected.
4. Set the feature flag to `0` and ensure the user is always redirected to the merchant checkout.
5. Set `_wcpay_feature_woopay_direct_checkout` to `1` and `_wcpay_feature_woopay_first_party_auth` to `0` and ensure the user is always redirected to the merchant checkout.

-------------------


**Post merge**
